### PR TITLE
Adds the 'twitter_username' attribute to NamedUser.

### LINKED ISSUE #1584: NamedUser object is missing the twitter_username property
The `twitter_username` property is available via the Github v3 REST API, as documented here: https://developer.github.com/v3/users/#get-a-user. However, it isn't parsed from the response to `get_user()`, so (e.g.) `Github().get_user("jacquev6").twitter_username` raises an Attribute error.

It looks pretty straightforward to submit a fix, and I'd be happy to do so if there's interest.

### PR DESCRIPTION
**Pull request by thisisdhaas**
Closes #1584 (It looked pretty easy, so I implemented my own fix).

I ran the `add_attribute.py` script to generate the updates, but noticed that the script doesn't update the `.pyi` files.

After my changes:
Screen Shot 2020-06-26 at 1 18 09 PM

### CODE REVIEW

File: tests/NamedUser.py [hunk 1]
```diff
@@ -110,6 +111,7 @@ def testAttributesOfSelf(self):
             self.user.suspended_at, datetime.datetime(2013, 8, 10, 7, 11, 7)
         )
         self.assertEqual(self.user.total_private_repos, 5)
+        self.assertEqual(self.user.twitter_username, None)
```
Comment (s-t-e-v-e-n-k) [inline tests/NamedUser.py:114]: ``self.assertIsNone(self.user.twitter_username)``

**Comment by thisisdhaas** [comment]
Awesome, ty! I made the fix.

Also shoutout to @marcua, who approves of this PR (and who helped me figure out that the field was missing).

**Comment by thisisdhaas** [comment]
@s-t-e-v-e-n-k I noticed that the checks failed—error seems due to a different test, maybe it's nondeterministically failing? This happened to me earlier, and I got tests to pass by simply re-running the checks with a force push...

**Comment by s-t-e-v-e-n-k** [comment]
@thisisdhaas Thanks for the fix! Bah, the failure is my fault -- I'll sort out a fix soonish and get you to rebase when it's landed.

**Comment by s-t-e-v-e-n-k** [comment]
My fix has landed, rebase onto master at your leisure.

**Comment by thisisdhaas** [comment]
...and done!

**Comment by s-t-e-v-e-n-k** [comment]
Thank you!

**Pull request merged**